packages/v4/patternfly-docs/content/design-guidelines/components/form/forms.md

```diff
@@ -32,10 +32,10 @@ Use forms to accomplish tasks reliant on user-inputted information like logging
 
 Most forms contain four basic elements:
 
-1. **Labels:** Indicate desired information for each field.
-2. **Text inputs:** Provide space for users to enter text.
-3. **Data inputs:** Allow users to enter information that is not freeform text using components like checkboxes, radio buttons, or dropdowns.
-4. **Buttons:** Enable a user to submit or exit a form.
+1. [**Labels:**](#Labels) Indicate desired information for each field.
```

Comment:
need to make sure all of these have a # after the bracket, and that they're in sentence case

so
#labels
#text-inputs
#data-inputs
#buttons